go.mod

```diff
@@ -11,54 +11,52 @@ require (
 	github.com/gammazero/keymutex v0.1.0
 	github.com/gammazero/radixtree v0.3.0
 	github.com/gammazero/workerpool v1.1.3
-	github.com/ipfs/go-cid v0.2.0
+	github.com/ipfs/go-cid v0.3.2
 	github.com/ipfs/go-log/v2 v2.5.1
 	github.com/ipld/go-storethehash v0.3.6
-	github.com/libp2p/go-libp2p-core v0.16.1
-	github.com/multiformats/go-multihash v0.1.0
+	github.com/multiformats/go-multihash v0.2.1
 	github.com/multiformats/go-varint v0.0.6
 	go.opencensus.io v0.23.0
-	golang.org/x/crypto v0.0.0-20211117183948-ae814b36b871
+	golang.org/x/crypto v0.0.0-20220525230936-793ad666bf5e
 )
 
-require lukechampine.com/blake3 v1.1.7
+require (
```

Comment:
why in separate require?

Reply:
That was just what `go mod tidy` did. I am looking for an explanation.